lib/situation_room/sites.ex

```diff
@@ -1,8 +1,27 @@
 defmodule SituationRoom.Site do
+
   use Ecto.Schema
+  import Ecto.Changeset
 
   schema "sites" do
     field :endpoint, :string
     field :name, :string
   end
+
+  # @spec changeset(
+  #         {map, map}
+  #         | %{
+  #             :__struct__ => atom | %{:__changeset__ => map, optional(any) => any},
+  #             optional(atom) => any
+  #           },
+  #         :invalid | %{optional(:__struct__) => none, optional(atom | binary) => any}
+  #       ) :: Ecto.Changeset.t()
+  def changeset(struct, params) do
+    struct
+    |> cast(params, [:endpoint, :name])
+    |> validate_required(:endpoint)
+    |> validate_length(:endpoint, min: 3)
+    # |> validate_format(:endpoint, ~r/./)
```

Comment:
if we aren't including this validation, let's removing it completely instead of leaving it in the commit commented out.